### ISSUE TITLE
Instabug.show() stops working after activity restart

### ISSUE THREAD

**Issue by mykhailo-gaidai** [Instabug/Instabug-Android#216]
### Steps to Reproduce the Problem
1. Instabug 8.6.0
2. Initialization in Application class
3. Single activity, multiple fragments app
4. Instabug is called with `Instabug.show()`
5. App is restarted with `startActivity(Intent.makeRestartActivityTask(requireActivity().componentName))` in order to apply `WindowManager.LayoutParams.FLAG_SECURE`
6. `Instabug.show()` is called
### Expected Behavior
Instabug window displayed as usual
### Actual Behavior
Instabug window is not displayed (nothing is logged in logcat) until app is restarted by the user (swipe away from the Recents screen)
### Instabug integration code
`Instabug.Builder(this, BuildConfig.INSTABUG_TOKEN)                .setInvocationEvents(InstabugInvocationEvent.NONE).build()
            BugReporting.setOptions(*FeatureFlags.instabugOptions)
            BugReporting.setAttachmentTypesEnabled(false, true, true, false)
            Instabug.setWelcomeMessageState(WelcomeMessage.State.DISABLED)`
### SDK Version
8.6.0
### Android Version
P-Q (28-29)
### Device Model
Pixel 3 (blueline)

**Comment by mzelzoghbi** [comment]
Hi @mykhailo-gaidai , Thanks for reporting this, We created a **SNAPSHOT** that should fix the issue, Can you please try it and tell us if the snapshot fixed the issue for you?
```
implementation 'com.instabug.library:instabug:8.6.3.2-SNAPSHOT'
```
you will need to add these lines to your app **build.gradle** before your dependencies
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```
This **SNAPSHOT** is completely safe to use and has no debug code at all.

**Comment by amrbrkt** [comment]
This was fixed in our latest version `8.7.0`.
